src/emulator/functionsEmulator.ts

```diff
@@ -395,18 +403,19 @@ export class FunctionsEmulator implements EmulatorInstance {
       return Promise.reject();
     }
 
+    const instance = result[1];
     const bundle = JSON.stringify({
       name: `projects/${projectId}/locations/_/functions/${definition.name}`,
```

Comment:
I'm pretty sure we store trigger payloads by unique name.

https://github.com/firebase/firebase-server/blob/master/firebase-server/firebase-core/src/main/scala/com/firebase/core/namespace/NamespaceActor.scala#L1559-L1560

 I'm not exactly sure how `definition` is determined. Will the `name` member be distinct for two different instances?

Reply:
Yeah function names are always unique, there's no way for me to export two functions with the same name but pointing to different instances.  So in this example above the names would be `triggerOne` and `triggerTwo`